clang/docs/UsersManual.rst

```diff
@@ -4401,13 +4413,18 @@ Execute ``clang-cl /?`` to see a list of supported options:
                               Instrument only functions from files where names don't match all the regexes separated by a semi-colon
       -fprofile-filter-files=<value>
                               Instrument only functions from files where names match any regex separated by a semi-colon
+      -fprofile-generate[=<dirname>]
+                              Generate instrumented code to collect execution counts into a raw profile file in <dirname>
+                              (overridden by LLVM_PROFILE_FILE env var)
       -fprofile-instr-generate=<file>
```

Comment:
Can we also update this line to match `-fprofile-generate`?

Reply:
Done